### ISSUE TITLE
Summary or Text is required.

### ISSUE THREAD

**Issue by Binyamse** [prometheus-msteams/prometheus-msteams#154]
**Describe the bug**
Some alerts work ok, meaning i get alerts on ms team but some alerts won't reach ms team with `"err":null,"level":"debug","response_message":"Summary or Text is required.","response_status":400,` error

**Version**
using helm with docker image `quay.io/prometheusmsteams/prometheus-msteams:v1.4.0`

**To Reproduce**
have this rule in your PrometheusRule

```yaml
- alert: kube_pod_container_terminated_error
        annotations:
         description: Pod {{$labels.pod}} in namespace {{$labels.namespace}} has a
           container terminated because of error
         summary: Pod {{$labels.pod}} in namespace {{$labels.namespace}} in error status
        expr: kube_pod_container_status_restarts_total - kube_pod_container_status_restarts_total offset 5m >= 1
         and ignoring(reason) kube_pod_container_status_last_terminated_reason{reason=~"error"}  > 0
        for: 10s
        labels:
          severity: critical
``` 
Exit one of your pods with error.  prometheus-msteams logs that there is 400 error. Some rules like `volume_claim_capacity_high_utilization` works with out issues. I am using the default tmp provided under helm chart.

```yaml
      - alert: volume_claim_capacity_high_utilization
        annotations:
          description: volume claim {{$labels.persistentvolumeclaim}} usage has exceeded
            80% of total capacity
          summary: '{{$labels.persistentvolumeclaim}} usage has exceeded 80% of total
            capacity.'
        expr: (kubelet_volume_stats_available_bytes / kubelet_volume_stats_capacity_bytes)
          < 0.2
        for: 5m
        labels:
          severity: critical
``` 
 

**Expected behavior**
Get alerts on MS

**Screenshots**

**Additional context**

**Comment by Knappek** [comment]
Hi @Binyamse, this is probably a duplicate of issue #153 . Feel free to re-open if that does not solve your issue.

**Comment by Knappek** [comment]
Oh, I just see that the chart's template is not in sync with the repo's default template. I will fix that.

**Comment by Binyamse** [comment]
Thank you very much @Knappek I will update to lates helm after issue is merge

**Comment by Knappek** [comment]
released in chart version `0.5.2`. Update your local repo with `helm repo update`.
